release/models/system/openconfig-system-management.yang

```diff
@@ -127,6 +127,93 @@ module openconfig-system-management {
     }
   }
 
+
+  grouping system-netconf-server-config {
+    description
+      "Configuration data for the netconf server/subsystem";
+
+    leaf netconf-transport-porotocol {
+      type enumeration {
+        enum SSH_v2 {
+          description
+            "Use SSH v2 only";
+        }
+        enum TCP {
+          description
+            "Use Transmission Control Protocol (TCP) only";
+        }
+        enum TLS {
+          description
+            "Use Transport Layer Security (TLS) only";
+        }
+        enum socket {
+          description
+            "Use socket connection only"
+      }
+      default SSH_v2;
+      description
+        "Enables the NETCONF server, using the SSHv2 transport. 
+        The NETCONF server is enabled by default."; 
+    }
+
+    leaf port {
+      type oc-inet:port-number;
+      default 830;
+      description
+        "TCP port on which the netconf should listen(rfc6242)";
+    }
+
+    leaf session-absolute-timeout {
+      type uint32;
+      units seconds;
+      description
+        "Absolute timeout in minutes";
+    }
+
+    leaf session-idle-timeout {
+      type uint32;
+      units seconds;
+      description
+        "Idle timeout in minutes";
```

Comment:
units = seconds yet description says minutes

Reply:
updated the description